lib/Form/EditContactGroup.php

```diff
@@ -36,9 +36,16 @@ public function __construct($vars, $contact)
         }
     }
 
-    public function renderActive($renderer, $vars, $action, $method)
+    public function renderActive(
```

Comment:
Parameters which are optional in parent should be optional in child.
Parameters which exist in parent should exist in child class. As they are optional, it does not hurt any existing calls.